src/remediations/fifi_2.js

```diff
@@ -28,9 +28,9 @@ const NONE = 'none';
 
 
 exports.checkRhcEnabled = async function () {
-    const rhcStates = await configManager.getCurrentState();
+    const rhcProfiles = await configManager.getCurrentProfile();
 
-    if (rhcStates.state.remediations === DISABLED) {
+    if (rhcProfiles.remediations === false) {
         return false;
```

Comment:
Now looking back at what I currently have it seems obvious that this can be simplified in to less lines.. 😆 But that is an interesting shortcut. I'll make a note of it for the future too 🙂